compiler/docgen.nim

```diff
@@ -159,6 +159,7 @@ proc getOutFile2(conf: ConfigRef; filename: RelativeFile,
   else:
     result = getOutFile(conf, filename, ext)
 
+var gIdx = 101
```

Comment:
This introduces global state and I don't understand why it's needed.

Reply:
Fixed